.github/workflows/manual-deploy.yml

```
@@ -0,0 +1,647 @@
+            # Create symbolic link to shared .env file
+            ln -sf ~/fin-invoice-ocr-team6/shared/.env ./.env
+            
+            # Create symbolic link to shared newrelic.js file
+            ln -sf ~/fin-invoice-ocr-team6/shared/newrelic.js ./newrelic.js
+            
+            # Create symbolic link to shared uploads directory
+            ln -sf ~/fin-invoice-ocr-team6/shared/uploads ./uploads
+            
+            # Run migrations
+            npx sequelize-cli db:migrate --env ${{ github.event.inputs.environment == "blue" && "production" || "staging" }}
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)